tests/test_util.py

```diff
@@ -30,3 +30,79 @@ def test_tartufo_clones_git_repo_into_temp_dir(self, mock_mkdtemp, mock_clone):
     def test_clone_git_repo_returns_path_to_clone(self, mock_mkdtemp):
         repo_path = util.clone_git_repo("https://github.com/godaddy/tartufo.git")
         self.assertEqual(repo_path, mock_mkdtemp.return_value)
+
+
+class OutputTests(unittest.TestCase):
+    @mock.patch("tartufo.util.click")
+    def test_echo_issues_echos_all_when_not_json(self, mock_click):
+        util.echo_issues([1, 2, 3, 4], False, "", "")
+        mock_click.echo.assert_has_calls(
+            (mock.call(1), mock.call(2), mock.call(3), mock.call(4)), any_order=False
+        )
+
+    @mock.patch("tartufo.util.click", new=mock.MagicMock())
+    @mock.patch("tartufo.util.json")
+    def test_echo_issues_outputs_proper_json_when_requested(self, mock_json):
+        issue_1 = scanner.Issue(scanner.IssueType.Entropy, ["foo"])
+        issue_2 = scanner.Issue(scanner.IssueType.RegEx, ["bar"])
+        util.echo_issues([issue_1, issue_2], True, "/repo", "/output")
+        mock_json.dumps.assert_called_once_with(
+            {
+                "project_path": "/repo",
+                "issues_path": "/output",
+                "found_issues": [
+                    {
+                        "issue_type": "High Entropy",
+                        "issue_detail": None,
+                        "diff": "No diff available.",
+                        "strings_found": ["foo"],
+                        "commit_time": None,
+                        "commit_message": None,
+                        "commit_hash": None,
+                        "file_path": None,
+                        "branch": None,
+                    },
+                    {
+                        "issue_type": "Regular Expression Match",
+                        "issue_detail": None,
+                        "diff": "No diff available.",
+                        "strings_found": ["bar"],
+                        "commit_time": None,
+                        "commit_message": None,
+                        "commit_hash": None,
+                        "file_path": None,
+                        "branch": None,
+                    },
+                ],
+            }
+        )
```

Comment:
Should we add a test for when as_json is false?

Reply:
Right up above. :) https://github.com/godaddy/tartufo/pull/57/files#diff-5832adf374920d75d4bf48e546367f53R36-R41